src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -523,6 +723,66 @@ Name|Type|Format|Description
 `errorMessage`|`string`|-|The error message, if any
 `systemId`|`string`|-|The External system Id that acknowledged the document
 
+### <a name="schema-paymentrequest"></a>Payment Request
+
+Name|Type|Format|Description
+---|---|---|---
+`systemId`|`string`|-|**Required** The external System ID that acknowledged the documents, it can be an empty String.
+`processingConfirmation`|`Array`|[Payment Confirmation request item](#paymentrequest)|Payment confirmations JSON request.
+
+### <a name="schema-paymentconfirmationrequest"></a>Payment Confirmation Request Item
+
+Name|Type|Format|Description
+---|---|---|---
+`docId`|`string`|-|The financial document Id to confirm.
+`processingStatusCode`|`string`|-|Payment status. VALUES: CP, PP (Completely Paid, Partially paid)
+`clearingDetails`|`array`|[Payment Documents details](#schema-paymentdocs)|Payment documents details, if any
+
+### <a name="schema-paymentdocs"></a>Payment Documents Details
+
+Name|Type|Format|Description
+---|---|---|---
+`clearingDate`|`string`|-|Date the payment got cleared
+`clearingAmount`|`double`|-|Amount cleared
+`clearingCurrency`|`string`|-|Currency value
+`receiver`|`array`|[Receiver details](#schema-receiver)|Receiver details.
+`clearingReference`|`array`|[Clearing reference details](#schema-clearing-reference)|Clearing reference details
+
+### <a name="schema-receiver"></a>Receiver Details
+
+Name|Type|Format|Description
+---|---|---|---
+`receiverId`|`string`|-|Receiver id
+`receiverName`|`double`|-|Name of receiver
+`receiverType`|`string`|-|Type of receiver
+
+### <a name="schema-clearing-reference"></a>Clearing reference Details
+
+Name|Type|Format|Description
+---|---|---|---
+`companyCode`|`string`|-|Company code
+`financialDocumentId`|`string`|-|Document id in ERP
+`fiscalYear`|`string`|-|Fiscal year
+`paymentRef`|`string`|-|Payment reference id.
+`paymentMethod`|`string`|-|Payment method
+
+### <a name="schema-paymentconfirmationresponse"></a>Payment Confirmation Response
+
+Name|Type|Format|Description
+---|---|---|---
+`PaymentConfirmationResponse`|`array`|[Payment Confirmation Response Item](#paymentresponse)|The JSON response body
+
+#### <a name="schema-PaymentConfirmationResponseItem"></a>PaymentConfirmationResponseItem
+
+Name|Type|Format|Description
+---|---|---|---
+`paymentConfirmationResult`|`string`|-|Payment confirmation result. Supported values are: SUCCESS, SYSTEM_ERROR_OCCURRED, NOT_POSTING_CONFIRMED, DOCUMENT_NOT_FOUND, FAILURE, NOOP
```

Comment:
The Swagger includes a field named paymentRef here, should that be in the markdown?

Reply:
done.